gutenTAG/api/bo.py

```diff
@@ -10,3 +10,4 @@
 from ..base_oscillations.sawtooth import sawtooth
 from ..base_oscillations.sine import sine
 from ..base_oscillations.square import square
+from ..base_oscillations.custom_input import custom_input  # not sure if this is correct. random_walk is imported similarly wihout the function being defined aas random_walk though.
```

Comment:
This is not correct and won't work. The `random_walk` function is defined, but was deleted within this PR.

Let's keep the new `CustomInput` BO out of the current _beta_ API. There is not much use for it anyway because one can just use `np.genfromtxt` or `pd.read_csv` manually with more control over it.